client/containers/Pub/PubDocument/PubWordCountButton.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useMemo, useState } from 'react';
+import { Popover, Button, Icon } from '@blueprintjs/core';
+
+import { Node } from 'prosemirror-model';
+
+require('./pubWordCountButton.scss');
+
+type Props = {
+	doc: Node;
+};
+
+const getWordAndCharacterCountsFromDoc = (node: Node) => {
+	const text = node.textBetween(0, node.content.size, ' ', ' ');
+	const words = text.split(' ').filter((word) => word !== '');
+	const wordCount = words.length;
+	const characterCount = words.reduce((a, x) => a + x.length, 0);
+	return [wordCount, characterCount];
+};
+
+const PubHeaderFormattingWordCountButton = (props: Props) => {
+	const { doc } = props;
+	const [wordCount, characterCount] = useMemo(() => getWordAndCharacterCountsFromDoc(doc), []);
+	const [open, setOpen] = useState(false);
+	const content = (
+		<div className="pub-word-count-button-popover-content">
+			<dl>
```

Comment:
ah, a rare one!

![image](https://user-images.githubusercontent.com/2208769/160435629-15d414d3-9664-435b-89af-58e44b0477fc.png)

Reply:
I wanna be the very best
Like no one ever was
learning tags is my real test
semantics is my cause